spec/amber/amber_spec.cr

```diff
@@ -37,6 +36,7 @@ describe Amber do
       end
```

Comment:
It seems like this should be a test for `Amber::Settings` rather than `Amber::Server.configure`

Reply:
Done.